src/runtime-rs/crates/resource/src/cpu_mem/initial_size.rs

```diff
@@ -105,7 +105,7 @@ impl InitialSizeManager {
             hv.cpu_info.default_vcpus = self.resource.vcpu as i32
         }
         if self.resource.mem_mb > 0 {
-            hv.memory_info.default_memory = self.resource.mem_mb;
+            hv.memory_info.default_memory += self.resource.mem_mb;
```

Comment:
I think this PR might be a duplicate, as this PR already did this operation https://github.com/kata-containers/kata-containers/pull/6876, I suggest we can get this merged ASAP.

Reply:
Sorry, I didn't see the message on the slack